common/go.mod

```diff
@@ -1,17 +1,22 @@
 // Copyright 2023 D2iQ, Inc. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-module github.com/d2iq-labs/capi-runtime-extensions/server
+module github.com/d2iq-labs/capi-runtime-extensions/common
```

Comment:
Renamed `server` module to `common` as it contains a lot more than just server code now.